src/frame-writer.cpp

```diff
@@ -44,19 +44,35 @@ void FrameWriter::init_hw_accel()
 
 void FrameWriter::load_codec_options(AVDictionary **dict)
 {
-    static const std::map<std::string, std::string> default_x264_options = {
-         {"tune", "zerolatency"},
-         {"preset", "ultrafast"},
-         {"crf", "20"},
+    using CodecOptions = std::map<std::string, std::string>;
+
+    static const CodecOptions default_x264_options = {
+        {"tune", "zerolatency"},
+        {"preset", "ultrafast"},
+        {"crf", "20"},
+    };
+
+    static const CodecOptions default_libvpx_options = {
+        {"cpu-used", "5"},
+        {"deadline", "realtime"},
     };
 
-    if (params.codec.find("libx264") != std::string::npos ||
-        params.codec.find("libx265") != std::string::npos)
+    static const std::map<std::string, const CodecOptions&> default_codec_options = {
+        {"libx264", default_x264_options},
+        {"libx265", default_x264_options},
+        {"libvpx", default_libvpx_options},
```

Comment:
Shouldn't there be also a line for `libvpx-vp9`?

Reply:
No, it's a substring search so `libvpx` also matches `libvpx-vp9`.
The original code did that to avoid listing codec variants like `libx264rgb` and I preserved the behavior.

Reply:
Ah, makes sense. Thanks for explaining.